src/world.rs

```diff
@@ -54,15 +55,13 @@ pub struct World {
 impl World {
     /// Create an empty world
     pub fn new() -> Self {
-        static ID: AtomicU64 = AtomicU64::new(1);
+        static ID: Mutex<u64> = Mutex::new(1);
```

Comment:
Let's add a comment here explaining the motivation (i.e. 32-bit ppc/mips support) for future reference.